Sources/Web3Auth/Types.swift

```diff
@@ -214,9 +215,10 @@ public struct W3AInitParams: Codable {
         chainNamespace = ChainNamespace.eip155
         useCoreKitKey = false
         mfaSettings = nil
-        sessionTime = 86400
+        sessionTime = 30 * 86400
         chainConfig = nil
         originData = nil
+        dashboardUrl = nil
```

Comment:
This shouldn't be nil because in manageMFA, since we forcefully make it non-nullable at line 373 in manageMFA function